axevent/send_event/app/send_event.c

```diff
@@ -120,26 +121,23 @@ static void declaration_complete(guint declaration, gdouble* value) {
 static guint setup_declaration(AXEventHandler* event_handler) {
     AXEventKeyValueSet* key_value_set = NULL;
     guint declaration                 = 0;
-    guint token                       = 0;
     gdouble start_value               = 0;
```

Comment:
Was it intentional to remove the token here?
The token is still in function description so that would need to be updated.

Also, make sure to check that the description in README.md is still correct.